libs/icon-button/src/index.tsx

```diff
@@ -8,31 +8,35 @@ import {
 import React from 'react';
 
 export type IconButtonProps = Omit<ChakraIconButtonProps, 'onClick' | 'aria-label'> & {
-  tooltip: Omit<TooltipProps, 'children'> & {
+  tooltip?: Omit<TooltipProps, 'children'> & {
     label: string;
   };
+  showTooltip?: boolean;
```

Comment:
I don't see the `showTooltip` prop in use

Reply:
Oh it's passed under `{... rest}`

But also - do we need a separate flag for show tool tip? Isn't it being nullable enough?

Reply:
definitely a leftover from previous implementation, removed